contrib/seeds/README.md

```diff
@@ -8,13 +8,19 @@ and remove old versions as necessary (at a minimum when SeedsServiceFlags()
 changes its default return value, as those are the services which seeds are added
 to addrman with).
 
-The seeds compiled into the release are created from sipa's DNS seed and AS map
-data. Run the following commands from the `/contrib/seeds` directory:
+The seeds compiled into the release are created from sipa's and achow101's DNS seed,
+virtu's crawler, and fjahr's community AS map data. Run the following commands from the
+`/contrib/seeds` directory:
 
 ```
 curl https://bitcoin.sipa.be/seeds.txt.gz | gzip -dc > seeds_main.txt
-curl https://bitcoin.sipa.be/asmap-filled.dat > asmap-filled.dat
+curl https://mainnet.achownodes.xyz/seeds.txt.gz | gzip -dc >> seeds_main.txt
+curl https://testnet.achownodes.xyz/seeds.txt.gz | gzip -dc > seeds_test.txt
+curl https://21.ninja/seeds.txt.gz | gzip -dc >> seeds_main.txt
+curl https://raw.githubusercontent.com/fjahr/asmap-data/main/latest_asmap.dat > asmap-filled.dat
```

Comment:
Mine is here if you want to add it:

https://luke.dashjr.org/programs/bitcoin/files/charts/seeds.txt

Reply:
nice, added!